determined-ai/determined
# feat: added authz to grpc master config endpoint

## Description
There was a hard check on the admin flag in `/api/v1/master/config` (GetMasterConfig), instead this check was moved to `CanGetMasterConfig`.




## Test Plan
- Confirm that admin users can access `/api/v1/master/config` 
- Confirm that non-admin users receive a permission denied error when attempting to access `api/v1/master/config`




## Commentary (optional)





## Checklist

  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.

## Ticket
DET-9629
